### PR TITLE
Simple calculator with exceptions Task 9

### PR DESCRIPTION
**Pull request by IrynaSemenenko**
It is a simple calculator with exceptions: NotSupportedOperation, InputMismatchException.

### CODE REVIEW

File: Exception/src/MethodsCalculator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.Scanner;
+
+public class MethodsCalculator {
+    Scanner sc = new Scanner(System.in);
+    double value = 0;
+    boolean invalidValue = true;
+    String operation;
+
+    public double setValue() {
+        while (invalidValue) {
```
Comment (Hmeliumm) [inline Exception/src/MethodsCalculator.java:11]: It's always true, please avoid such loops

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Exception/src/MethodsCalculator.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class MethodsCalculator {
+    Scanner sc = new Scanner(System.in);
```
[the remark this answers is not in the document]
Comment (IrynaSemenenko) [reply Exception/src/MethodsCalculator.java]: Done

File: Exception/src/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+public class Calculator {
+    public static void main(String[] args) {
```
[the remark this answers is not in the document]
Comment (IrynaSemenenko) [reply Exception/src/Calculator.java:2]: Done

File: Exception/src/MethodsCalculator.java (hunk 1 above)
Comment (IrynaSemenenko) [reply Exception/src/MethodsCalculator.java:14]: I understand, but my program doesn't work without it, and I didn't find a better solution for it. So I didn't change it.

Comment (Hmeliumm) [reply Exception/src/MethodsCalculator.java:14]: OK

**Review by Hmeliumm — APPROVED** [review]

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**